### ISSUE TITLE
Issue in English translation

### ISSUE THREAD

**Issue by mbousq** [cboxdoerfer/fsearch#355]
I think there is an error in the english po :)

| Current (error) | New (my guess) |
|-----------------|----------------|
| One Filesystem  | Update path    |

**Comment by cboxdoerfer** [comment]
No, it's not an error. But maybe it's not the best way to describe this feature, I just copied the wording from `rsync`:

> --one-file-system, -x    don't cross filesystem boundaries

because I couldn't think of a better name. Basically it means that FSearch will stay on the file system of the particular folder during scans. That's especially useful when you have network shares mounted in some sub folders and don't want FSearch to traverse into them.

**Comment by mbousq** [comment]
Got it :). I got confused by the "help" message actually. On the UI, the columns are: Active, Path and One Filesystem but the description has Active, Path and Update instead.

> #: src/fsearch_preferences.ui:1645
> msgid ""
> "<b>Included folders:</b>\n"
> "\n"
> "                      This list contains all folders you want to include in "
> "the database.\n"
> "\n"
> "                      • <b>Active:</b> If enabled, the path will included in "
> "the database to be searched in.\n"
> "                      • <b>Path:</b> The full path to be included.\n"
> "                      • <b>Update:</b> This setting is useful if you want to "
> "exclude certain paths from\n"
> "                      being\n"
> "                      updated <i>(e.g. a path to a slow USB drive which "
> "doesn't change often)</i>\n"
> "                    "

**Comment by cboxdoerfer** [comment]
Ah yes, the help text is simply outdated. I'll fix it, thx.
